### PR TITLE
Add a section about integrating collaboration plugins in the angular app

### PR DESCRIPTION
**Pull request by ma2ciek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Docs: Added a section about integrating collaboration plugins in the angular app. Closes ckeditor/ckeditor5-angular#6.

---

### Additional information

*For example – encountered issues, assumptions you had to make, other affected tickets, etc.*

### CODE REVIEW

**Comment by Reinmar** [comment]
I like the text 👍

I'm removing @mlewand and adding @AnnaTomanek for proofreading.

File: docs/builds/guides/frameworks/angular.md [hunk 1]
```diff
@@ -128,6 +128,18 @@ Note that to allow importing JavaScript files without providing their correspond
 	If you cannot set the target higher than `es5`, try to set `"buildOptimizer": false` which will produce a bigger, but correct production build.
 </info-box>
 
+### Using editor with collaboration plugins
```
Comment (AnnaTomanek) [inline docs/builds/guides/frameworks/angular.md]:
```suggestion
### Using the editor with collaboration plugins
```

File: docs/builds/guides/frameworks/angular.md [hunk 2]
```diff
@@ -128,6 +128,18 @@ Note that to allow importing JavaScript files without providing their correspond
...
+
+The easiest way to integrate collaboration plugins in the Angular application is to create a custom build first and later import it from the Angular application - see [Using a custom CKEditor 5 build](#using-a-custom-ckeditor-5-build).
```
Comment (AnnaTomanek) [inline docs/builds/guides/frameworks/angular.md]:
```suggestion
The easiest way to integrate {@link features/collaboration collaboration plugins} in an Angular application is to create a custom build first and later import it from the Angular application &mdash; see [Using a custom CKEditor 5 build](#using-a-custom-ckeditor-5-build).
```

File: docs/builds/guides/frameworks/angular.md [hunk 3]
```diff
@@ -128,6 +128,18 @@ Note that to allow importing JavaScript files without providing their correspond
...
+
+For such scenario we provide a few ready-to-use integrations featuring the collaborative editing in Angular applications:
```
Comment (AnnaTomanek) [inline docs/builds/guides/frameworks/angular.md]:
```suggestion
For such scenario we provide a few ready-to-use integrations featuring collaborative editing in Angular applications:
```

File: docs/builds/guides/frameworks/angular.md [hunk 4]
```diff
@@ -128,6 +128,18 @@ Note that to allow importing JavaScript files without providing their correspond
...
+
+- [CKEditor 5 with real-time collaboration features](https://github.com/ckeditor/ckeditor5-collaboration-samples/tree/master/real-time-collaboration-for-angular)
+
+- [CKEditor 5 with the track changes feature](https://github.com/ckeditor/ckeditor5-collaboration-samples/tree/master/track-changes-for-angular)
+
+It is not mandatory to build applications on top of the above samples, however, they should provide a feeling of how to start.
```
Comment (AnnaTomanek) [inline docs/builds/guides/frameworks/angular.md]:
```suggestion
It is not mandatory to build applications on top of the above samples, however, they should help you get started.
```

**Comment by AnnaTomanek** [comment]
This should actually land in `stable`, not `master` as it applies to what's currently available. Thanks for the update!

**Comment by ma2ciek** [comment]
> This should actually land in stable, not master as it applies to what's currently available. Thanks for the update!

Right, Thanks for the review!

**Comment by ma2ciek** [comment]
@pomek, would you like to add a similar paragraph to the React integration as well?

**Pull request merged**